src/main/kotlin/no/nav/familie/ef/sak/behandling/BehandlingRepository.kt

```diff
@@ -164,6 +164,12 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
         FROM gjeldende_iverksatte_behandlinger gib 
             JOIN person_ident pi ON gib.fagsak_person_id=pi.fagsak_person_id
         WHERE gib.stonadstype=:stønadstype AND (vedtakstidspunkt < ('now'::timestamp - '3 month'::interval) OR arsak IN ('MIGRERING', 'G_OMREGNING'))
+        EXCEPT
+        (SELECT pi.ident
+         FROM fagsak
+                  JOIN behandling ON fagsak.id = behandling.fagsak_id
```

Comment:
Ser bra ut, men `finnPersonerMed [AKTIV] Stønad...`
Hvis du fikk innvilget for 1 år siden, men stønaden gikk ut i januar - vil ikke disse også bli med i utplukket her?
Ønsker vi egentlig å finne de som har innvilget penger nå/frem i tid, eller er de gamle også av interesse? 🤔